qc2md.py

```diff
@@ -100,6 +100,18 @@ def parse_args() -> argparse.Namespace:
         default=True,
         help="Display a picker interface if there are multiple matching reference lines (default: %(default)s)",
     )
+    parser.add_argument(
+        "--nondialogue-categories",
```

Comment:
I would put `--standalone-categories` as the first option here, as it applies first in the process; this is the only place in code where `nondialogue_categories` is handled first.